app/components/applications-app-services/locals.tf

```diff
@@ -15,6 +15,7 @@ locals {
         FILE_UPLOADS_PATH                   = "/opt/app/uploads"
         GOOGLE_ANALYTICS_ID                 = var.google_analytics_id
         HOST_URL                            = "https://${var.applications_service_public_url}/"
+        REDIS_CONNECTION_STRING             = local.secret_refs["applications-service-redis-cache-connection-string"]
```

Comment:
This will reference the Key Vault secret name below with default value `<enter_value>`. If you want to reference the secret created in this PR then we would use the format: `"@Microsoft.KeyVault(SecretUri=${var.key_vault_uri}secrets/applications-service-redis-cache-connection-string/)"` but would probably want to pass in the secret name `applications-service-redis-cache-connection-string` as a variable to ensure the dependency graph correctly creates the secret before adding the Key Vault reference to the App Service, e.g.: `"@Microsoft.KeyVault(SecretUri=${var.key_vault_uri}secrets/${var.applications_service_redis_connection_string}/)"`